_posts/2018-12-17-the-lazy-applicative-functor.html

```diff
@@ -57,10 +57,10 @@ <h3 id="c04ca371d39847cc86a90240a7f53745">
 <span style="color:#2b91af;">Lazy</span>&lt;<span style="color:blue;">string</span>&gt;&nbsp;s&nbsp;=&nbsp;f.Apply(c).Apply(i);</pre>
 	</p>
 	<p>
-		Notice that you can chain the various overloads of <code>Apply</code>. In the above example, you have a lazy function that takes a <code>char</code> and an <code>int</code> as input, and returns a <code>string</code>. It could, for instance, be a function that invokes <a href="https://docs.microsoft.com/en-us/dotnet/api/system.string.-ctor?view=netframework-4.7.2#System_String__ctor_System_Char_System_Int32_">the equivalent <code>string</code> constructor overload</a>.
+		Notice that you can chain the various overloads of <code>Apply</code>. In the above example, you have a lazy function that takes a <code>char</code> and an <code>int</code> as input, and returns a <code>string</code>. It could, for instance, be a function that invokes <a href="https://docs.microsoft.com/en-us/dotnet/api/system.string.-ctor?view=netframework-4.7.2#system-string-ctor(system-char-system-int32)">the equivalent <code>string</code> constructor overload</a>.
 	</p>
 	<p>
-		Calling <code>f.Apply(c)</code> uses the overload that takes a <code>Lazy&lt;Func&lt;T1, T2, TResult&gt;&gt;</code> as input. The return value is a <code>Lazy&lt;Func&lt;int, string&gt;&gt;</code>, which the first <code>Apply</code> then picks up, to return a <code>Lazy&lt;string&gt;</code>.
+		Calling <code>f.Apply(c)</code> uses the overload that takes a <code>Lazy&lt;Func&lt;T1, T2, TResult&gt;&gt;</code> as input. The return value is a <code>Lazy&lt;Func&lt;int, string&gt;&gt;</code>, which the second <code>Apply</code> then picks up, to return a <code>Lazy&lt;string&gt;</code>.
```

Comment:
I can see that I phrased this in an unclear manner, but I meant _the first `Apply` **overload**_.

Reply:
I see, it kind of tricked me as the "first" Apply call had already been described, that seemed like the "second" call. As before, I can revert this change as well, or if you have a better wording, don't know.

Reply:
"the first `Apply` overload" would work, I think.